### DISCUSSION TITLE
Usage outside Livewire?

### DISCUSSION THREAD

**Discussion by mattkingshott** [livewire/flux#865] [Feature Requests] [answered]
I’m looking at using Flux for an upcoming project and I was just wondering if it is possible to use Flux with just Blade and Alpine?

In other words, do you have to use flux within the confines of a Livewire component?

**Comment by joshhanley** [comment]
@mattkingshott at this stage, we officially say it needs to be used inside Livewire. Some of the components should work outside of Livewire, but it's not guaranteed. For example the editor component won't work outside of Livewire at the moment because it's using `@scripts` which is a Livewire directive.

If you do try it, we'd love to know what you find that does and does not work 🙂

**Reply by mattkingshott** [reply]
Thanks Josh. 

I wonder…

If I were to install and set up Livewire in my app / on my layout file, but not actually use Flux components within a Livewire component, would that work?

in other words, Flux would have access to Livewire scripts, styles etc. but I would just be using the flux components within traditional blade views.

**Reply by joshhanley** [reply, marked as answer]
@mattkingshott as mentioned above, some of the components will work outside of Livewire (they don't depend on Livewire at all), but some like the editor component do and needs to be inside a Livewire component to function (as `@scripts` doesn't function without a Livewire component context).

You shouldn't need Livewire's scripts and styles though, as most (if not all) of Flux's components don't depend on them, as they have their own dedicated JS. So I'd start without them and see how you go.

**Reply by mattkingshott** [reply]
Great. Thank you!
